README.md

```diff
@@ -8,7 +8,7 @@ A command line tool and Python library to support your accounting
 process.
 
 1. extracts text from PDF files using different techniques, like
-   `pdftotext`, `text`, `pdfminer` or OCR -- `tesseract`, `tesseract4` or
+   `pdftotext`, `pdfminer` or OCR -- `tesseract`, or
```

Comment:
Please leave `text` here :)

Reply:
Fine by me ;) Will fix this merge conflict